### ISSUE TITLE
Scaling for SVG areas

### ISSUE THREAD

**Issue by Klaus-Tockloth** [mapsforge/vtm#215]
According to the XSD scaling for areas isn't supported yet:

```
<xs:complexType name="area">
    <xs:attribute name="cat" type="xs:string" use="optional" />
    <xs:attribute name="id" default="0" type="xs:string" use="optional" />
    <xs:attribute name="use" default="0" type="xs:string" use="optional" />
    <xs:attribute name="src" type="tns:src" use="optional" />
    <xs:attribute name="fill" default="#000000" type="tns:color" use="optional" />
    <xs:attribute name="stroke" default="#00000000" type="tns:color" use="optional" />
    <xs:attribute name="stroke-width" default="0" type="tns:nonNegativeFloat" use="optional" />
    <xs:attribute name="fade" default="-1" type="xs:integer" use="optional" />
    <xs:attribute name="blend" default="-1" type="xs:integer" use="optional" />
    <xs:attribute name="blend-fill" default="#000000" type="tns:color" use="optional" />
    <xs:attribute name="mesh" default="false" type="xs:boolean" use="optional" />
</xs:complexType>
```

Mapsforge has options for scaling:

```
<rule e="way" k="leaf_type" v="needleleaved">
   <area src="file:/patterns/nadelwald.svg" symbol-scaling="size" symbol-width="32" />
</rule>
```

Example:
![bildschirmfoto 2016-10-29 um 21 02 50](https://cloud.githubusercontent.com/assets/5493128/19832101/25d52944-9e1b-11e6-8233-38371959a251.jpg)

Please treat / see this issue as a feature request.

**Comment by devemux86** [comment]
See my [comment](https://github.com/mapsforge/vtm/issues/214#issuecomment-257109728) in #214.
Related to #74.

**Comment by devemux86** [comment]
I'm closing this, as it will be implemented for all SVG resources in #214.
